code_snippets/java/src/main/java/develop/ServiceCommunication.java

```diff
@@ -9,8 +9,7 @@ private void requestResponseService(Context ctx) {
         String request = "";
 
         // <start_request_response_service>
-        MyServiceClient.ContextClient client = MyServiceClient.fromContext(ctx);
-        String response = client.myHandler(request).await();
+        MyServiceClient.fromContext(ctx).myHandler(request).await();
```

Comment:
If in the java case you wanna collapse this in a oneliner, then do it in kotlin too.